numpy/core/tests/test_multiarray.py

```diff
@@ -6734,16 +6733,17 @@ def test_ctypes_is_available(self):
         test_arr = np.array([[1, 2, 3], [4, 5, 6]])
 
         self.assertEqual(ctypes, test_arr.ctypes._ctypes)
-        assert_equal(test_arr.shape, (2, 3))
+        assert_equal(list(test_arr.ctypes.shape), (2, 3))
```

Comment:
`assert_equal(list(...), some_tuple)` is a little jarring - maybe replace `list` with `tuple`?